src/main/java/practice/StreamPractice.java

```diff
@@ -1,20 +1,34 @@
 package practice;
 
-import java.util.Collections;
+import java.util.Arrays;
+import java.util.Comparator;
 import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 import model.Candidate;
+import model.Cat;
 import model.Person;
 
 public class StreamPractice {
+    private static final String NUMBER_SEPARATOR = ",";
     /**
      * Given list of strings where each element contains 1+ numbers:
      * input = {"5,30,100", "0,22,7", ...}
      * return min integer value. One more thing - we're interested in even numbers.
      * If there is no needed data throw RuntimeException with message
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
+
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(string -> string.split(NUMBER_SEPARATOR))
+                .flatMap(Arrays::stream)
+                .map(Integer::parseInt)
+                .filter(number -> number % 2 == 0)
+                .min(Comparator.naturalOrder()).orElseThrow(()
+                        -> new RuntimeException("Can't get min value from list: "
+                        + "< Here is our input 'numbers' >"));
```

Comment:
May be like this it would be better)
```suggestion
                        -> new RuntimeException("Can't get min value from list: "
                        + numbers));
```